### ISSUE TITLE
value is not changing

### ISSUE THREAD

**Issue by airrooms** [loebi-ch/jquery-clock-timepicker#41]
Hi I used onAudjust and also try onChange to change the value. but value on changing. can you plaese help.

	onAdjust: function(newVal, oldVal) {
  return   $(this).attr('value');  
	}

**Comment by loebi-ch** [comment]
Hi @RoomLea 

What exactly do you try to achieve? I cannot follow you...

```onAdjust``` is an event that's fired every time the clock's value is changing
```onChange``` is an event that's fired every time the clock's value has been changed

If you want to set the clock's value, you have to use the following method:

```$('.your-time-field').clockTimePicker('value', '08:00');```

**Comment by airrooms** [comment]
![2022-02-23_1-41-38](https://user-images.githubusercontent.com/100233858/155216065-a395a086-443c-4ec8-8d9b-2ec25547d560.png)

check this image value still same how to get new time value?

**Comment by loebi-ch** [comment]
The value in HTML is only the initial value for the ClockTimePicker.

You can register the ```onchange``` event in HTML:

```<input class="time" type="text" ... onchange="console.log('Time changed to: ' + this.value)" />```

or you can register the ```onChange``` or ```onAdjust``` event in the ClockTimePicker's configuration:

```
$('.time').clockTimePicker({
   onChange: function(newVal, oldVal) { console.log('Value changed from ' + oldVal + ' to ' + newVal + '.'); },
   onAdjust: function(newVal, oldVal) { console.log('Value adjusted from ' + oldVal + ' to ' + newVal + '.'); }
});
```

or you can simply call:

```$('.time').val()```

**Comment by loebi-ch** [comment]
I've just released a new version (2.6.0) that also updates the HTML value attribute upon time change.
